bftengine/src/bftengine/PersistentStorageImp.hpp

```diff
@@ -99,14 +99,18 @@ enum DescMetadataParameterIds {
   LAST_NEW_VIEW_DESC
 };
 
+const uint32_t clientsDataDescNum = LAST_NEW_VIEW_DESC + 1;
+const uint32_t replicaSpecificInfoMaxSize = 64 * 1024;  // 64KB
+
 typedef unique_ptr<MetadataStorage::ObjectDesc[]> ObjectDescUniquePtr;
 
 class PersistentStorageImp : public PersistentStorage {
  public:
   static constexpr auto kMaxUserDataSizeBytes = 256;
 
  public:
-  PersistentStorageImp(uint16_t numReplicas, uint16_t fVal, uint16_t cVal);
+  PersistentStorageImp(
+      uint16_t numReplicas, uint16_t fVal, uint16_t cVal, uint64_t numClients, uint64_t numOfClientBatch);
```

Comment:
numClients could not include RO replicas, split the parameter to two or give a more descriptive name.

Reply:
I changed it to include all principles ids (including replicas).
The name has changed accordingly